rocprim/CMakeLists.txt

```diff
@@ -53,27 +59,30 @@ set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR})
 # does not support header-only libraries (INTERFACE targets)
 rocm_install_targets(
   TARGETS rocprim rocprim_hip
-  PREFIX rocprim
 )
 
 rocm_install(
   DIRECTORY
     "include/"
     "${PROJECT_BINARY_DIR}/rocprim/include/"
```

Comment:
Why is this still `"${PROJECT_BINARY_DIR}/rocprim/include/"`?

Reply:
it was include only in the last version, Updated in the latest version with new structure